www/scipylib/building/windows.rst

```diff
@@ -32,6 +32,109 @@ information than this document will provide.
 .. _OpenBLAS: https://github.com/xianyi/OpenBLAS
 .. _LAPACK: http://www.netlib.org/lapack/
 
+
+Building the Released SciPy
+---------------------------
+
+This section provides the step-by-step process to build the released scipy. If you want
+to build completely from source, you should estimate at least three hours to build all
+libraries and compile SciPy. Feel free to stop and inspect any step at any time, but
+we're mostly just going to breeze through this. More in-depth documentation is provided 
+below.
+
+First, download all of the following software concurrently:
+
+1) MinGW-w64 from https://mingw-w64.org
+2) Microsoft Visual Studio 2015 or 2017 Community Edition (available from Microsoft)
+3) git from https://git-scm.org/
+4) MSYS2 from http://www.msys2.org/
+5) Python **with pip** from https://python.org/
+
+While that's downloading let's outline our plan of action. We're going to:
+
+1) Build OpenBLAS using the MinGW-w64 compiler suite, using the build system from MSYS2
+2) Copy the associated libraries to the `Python\\Lib` directory
+3) Build SciPy with a hybrid MinGW-w64 toolchain
+
+Once git installed, open a terminal in your favorite directory and type the following:
+
+.. code:: shell
+   git clone https://github.com/matthew-brett/build-openblas.git
+   cd build-openblas
+   git submodule update --init --recursive
+
+Now let's build OpenBLAS. Open a MSYS2 terminal and change directories to the
+`build-openblas` folder. To make sure that we're ready to build, type the following in
+the terminal:
+
+.. code:: shell
+   git
```

Comment:
I guess this is git from the git installer, present here because it is on the Windows PATH?